lib/mobilize_america_client/client/events.rb

```diff
@@ -33,12 +34,40 @@ def organization_events(organization_id:, timeslot_start: nil, timeslot_end: nil
           end
         end
 
+        unless event_campaign_id.nil?
+          params[:event_campaign_id] = event_campaign_id.to_i
+        end
+
+        unless tag_ids.nil? || tag_ids.empty?
+          params[:tag_id] = tag_ids
+        end
+
+        unless event_types.nil? || event_types.empty?
+          params[:event_types] = event_types
+        end
+
+        unless is_virtual.nil?
+          params[:is_virtual] = is_virtual ? 'true' : 'false'
```

Comment:
It's probably good enough? I guess it would be unexpected if you tried to pass `false` in as a string.... but not sure the gem needs to be super smart.